### DISCUSSION TITLE
How to add a background image to a post?

### DISCUSSION THREAD

**Discussion by sahashirshendu** [adityatelange/hugo-PaperMod#988] [Q&A] [answered]
Hello, in the [website](https://teilt.github.io/posts/test) I would like to add a static background image for the whole post (something like [this](https://presision21.weebly.com/schedule.html)). It would be very helpful if anyone could give any idea how to do so. Thanks in advance.

**Comment by sahashirshendu** [comment, marked as answer]
I figured it out:
```html
{{ if .Params.imgbg -}}
  {{- range first 1 .Params.imgbg -}}
	<style>.post-content {background-image: url('{{. | absURL}}');}</style>
  {{- end -}}
{{- end -}}
```
and then giving the image path in `imgbg` in post header ...
Sorry for the inconvenience.

**Reply by JingyiWu-codespace** [reply]
hello, I would like to ask where to put the code block. on the boseof.html?  thank you
